src/Mod/Part/App/TopoShapeMapper.cpp

```diff
@@ -0,0 +1,75 @@
+#include "TopoShapeMapper.h"
+
+namespace Part
+{
+
+void ShapeMapper::expand(const TopoDS_Shape &d, std::vector<TopoDS_Shape> &shapes)
+{
+    if (d.IsNull()) return;
+    for(TopExp_Explorer xp(d, TopAbs_FACE);xp.More();xp.Next())
+        shapes.push_back(xp.Current());
```

Comment:
Our clang-format file specifies that there should be curly braces around all loops and conditionals, even if they are only a single line. It's also got things to say about spacing around semicolons in for loops, etc. -- are you using an editor that can be configured to auto-apply it?

Reply:
Yep, VSCode will do that.   I'll set it up.  I had been shying away from modifying copied in code as much as possible; I'll bring it up to format instead.